modules/smiley.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+'''
+smiley.py - Smiley Module
+Copyright 2014 Sujeet Akula (sujeet@freeboson.org)
+Licensed under the Eiffel Forum License 2.
+
+More info:
+ * jenni-misc: https://github.com/freeboson/jenni-misc/
+ * jenni: https://github.com/myano/jenni/
+ * Phenny: http://inamidst.com/phenny/
+'''
+
+import io, random, os
+
+def smiley(jenni, input):
+    '''.smiley -- print a random smiley'''
+
+    with io.open(os.getcwd() + '/modules/smileys.txt', 'r', encoding='utf-16-le') as f:
```

Comment:
We're really trying to avoid having these files all over the place. This will be resolved by https://github.com/myano/jenni/issues/107 but in the meanwhile can you move this into a standard place (like default.py)?